cmd/earthly/app/run.go

```diff
@@ -181,11 +182,15 @@ func (app *EarthlyApp) run(ctx context.Context, args []string) int {
 		grpcErr, grpcErrOK := grpcerrors.AsGRPCStatus(err)
 		hintErr, hintErrOK := getHintErr(err, grpcErr)
 		var paramsErr *params.Error
+		var autoSkipErr *inputgraph.Error
 		switch {
 		case hintErrOK:
 			app.BaseCLI.Logbus().Run().SetGenericFatalError(time.Now(), logstream.FailureType_FAILURE_TYPE_OTHER, hintErr.Message())
 			app.BaseCLI.Console().HelpPrintf(hintErr.Hint())
 			return 1
+		case errors.As(err, &autoSkipErr):
+			app.BaseCLI.Logbus().Run().SetGenericFatalError(time.Now(), logstream.FailureType_FAILURE_TYPE_SYNTAX, inputgraph.FormatError(err))
```

Comment:
is there any hint we should add here?

Reply:
Not at the moment, but perhaps we can improve the hints down the road.

Reply:
It would be nice for this to have its own failure type. In any case, it's not critical.